tests/test_templates.py

```diff
@@ -128,8 +127,14 @@ def test_table(self):
     </div>
 </body>
 </html>"""
-
         html = render_response(response=response, status_code=200)
-        expected = [line.strip() for line in expected.splitlines(1)]
-        html = [line.strip() for line in html.splitlines(1)]
+        expected = self._strip_whitespace(expected)
+        html = self._strip_ws(html)
         self.assertEqual(expected, html, "Table response incorrect")
+
+    _strip_ws(self, in_text):
```

Comment:
`def`?

Also, `_strip_ws` is not a good name since it's converting a string into a list with whitespace stripping as a bonus feature.